server_plugin/server.py

```diff
@@ -208,6 +223,9 @@ def stop(vca_client, **kwargs):
 @operation
 @with_vca_client
 def delete(vca_client, **kwargs):
+    if ctx.node.properties.get('use_external_resource'):
```

Comment:
note that even when using an external resource, the "delete" operation should clean up the runtime properties stored on the node instance.